CIP-0165/namespaces/utxo.cddl

```diff
@@ -0,0 +1,118 @@
+; This file was auto-generated from huddle. Please do not modify it directly!
+
+; entry in scls file
+generic_record<a0,b0> = {key : a0, value : b0}
+
+; entry in utxo namespace
+record_entry = generic_record<tx_in,tx_out>
+
+
+tx_in = [hash32, uint .size 2]
+
+hash32 = bytes .size 32
+
+tx_out = [0, shelley_tx_out// 1, babbage_tx_out]
```

Comment:
It's extremely inefficient to store TxOuts in the ledger state in CBOR format.
Earlier this year we introduced a different format for it (which is also used by UTxOHD), that has cut snapshot creation time in half and alleviated the problem of occasional block production being missed because of GC kicking in at the wrong time and taking too long. See https://github.com/IntersectMBO/ouroboros-consensus/issues/868